### DISCUSSION TITLE
Extra lines in ParaView visualisation

### DISCUSSION THREAD

**Discussion by drozp94** [OpenRadioss/OpenRadioss#484] [Users]
Hello,
does anybody know how to get rid of some extra lines after exporting the open radios animation file to vtk.
When the vtk file is opened in ParaView, some extra lines are are shown for each part and each line is pointing to the same center.
ParaView version used by me is: 5.10.0-RC1.
I've attached the example below:
![image](https://user-images.githubusercontent.com/113352419/190896957-906c425e-656a-4406-a3db-381c290bef21.png)

Thank you!
Dima

**Comment by SebVilleneuve** [comment]
Hi Dima,

It's certainly because you havesome rigif bodies in the model.

Regards,
Sébastien

**Reply by drozp94** [reply]
Hello Sébastien,
thank you for your response!
Do you know some way to hide those lines ?

Thank you!
Dima

**Reply by sebastienVilleneuve** [reply]
Hello, 

a solution is to identify the Part_Id of the lines , & use threshold filters to hide this.

Sébastien

**Comment by drozp94** [comment]
Hello Sébastien,

unfortunately it is not working,
I did a one more check and I can say that only deformable parts (O-ring in my case) have those lines,
please see the image below:
![image](https://user-images.githubusercontent.com/113352419/190956459-573a1245-9170-4b59-b1b1-ae9d8be6ae7a.png)

Filtering the Part ID will not solve the problem, I will check if some newer version of ParaView will solve this issue.

Dima

**Reply by drozp94** [reply]
Hello again,

In the latest ParaView the lines are still show,
but I've found a way to get rid of them, more exactly I've observed that those lines have Element ID of 0, so using the threshold filter I was able to solve my issue!
![image](https://user-images.githubusercontent.com/113352419/190959031-fa43ccce-2b89-495f-ba24-30a89198e6f3.png)


Thank you again  Sébastien for hints and discussion!
Dima

**Comment by sebastienVilleneuve** [comment]
as a workaround for this model , you can select the 1d elems , then apply extractSelection filter , & reverse the selection.

![image](https://user-images.githubusercontent.com/100404030/190959039-9d50fe18-3670-4b44-babb-12aa3e31a137.png)


Sébastien

**Reply by drozp94** [reply]
Nice, it would be another option!

Dima
